sigproc/src/OmnibusSigProc.cxx

```diff
@@ -790,7 +790,7 @@ void OmnibusSigProc::init_overall_response(IFrame::pointer frame)
 
     // Convert each average FR to a 2D array
     for (int iplane = 0; iplane < 3; ++iplane) {
-        auto arr = Response::as_array(fravg.planes[iplane], fine_nwires, fine_nticks);
+        auto arr = Response::as_array(*(fravg.plane(iplane)), fine_nwires, fine_nticks);
```

Comment:
This is a problem I only appreciate after seeing it as a code change.  See my comment in #322.